views/random-beers.hbs

```diff
@@ -0,0 +1,10 @@
+
+        <li>
+            {{#each beer}}
+            <img src={{image_url}}></br>
+            <h1>{{name}}</br></h1>
+            {{description}}</br>
+            <b>{{tagline}}</br></b>
+        </li>
+        {{/each}}
+    
```

Comment:
Good solutions so far! If you find the time, try to implement the partial for the beerCard. 🙂